src/layouts/AppBase/AppBase.js

```diff
@@ -35,6 +35,10 @@ function AppBase(props) {
 
 AppBase.propTypes = {
   children: PropTypes.node.isRequired,
+  // eslint-disable-next-line react/no-unused-prop-types
+  headerColor: PropTypes.string,
+  // eslint-disable-next-line react/no-unused-prop-types
+  headerMode: PropTypes.string,
```

Comment:
Not sure if these are needed in `AppBase` as the `App` layout component is meant to not pass them down as it's the `App` component that holds the "Header" component which uses these props. @alexanderflink